packages/components/src/local/organisms/view-as/types/props.d.ts

```diff
@@ -0,0 +1,8 @@
+export default interface PropTypes {
+    // current set of forces
```

Comment:
stop these being optional.  If we're going to have a `view-as` component, then we'll have the data